packages/codemods/1.x-2.x/__testfixtures__/computed-decorator.input.js

```diff
@@ -1,15 +1,15 @@
 /* eslint-disable */
-import {connect as Cerebral} from 'cerebral/react';
-import {limitedList} from "./limitedList";
-import {anotherComputed} from "./anotherComputed";
+import {connect as Cerebral} from "cerebral/react"
```

Comment:
Does not work. Seems they are also excluded on file base.
Will leave them as they're not part of code base.